### ISSUE TITLE
Тест: Defect - "З чого складається Description у багу?" question - “Оточення (Environment)” is missing in the answer options

### ISSUE THREAD

**Issue by DmitriiNasonov** [scholokov/qax-portal-2#1978]
**Steps of reproducing**
1. Open https://www.qax-camp.com.ua/courses/q6-basic/lessons/q6-defect/quizzes/deffect/
2. Open the "З чого складається Description у багу?" question
3. Pay attention to the answer options

**Actual Result**
“Оточення (Environment)” is missing in the answer options

**Expected Result**
“Оточення (Environment)” is present in the answer options

**Additional information**
[Page 15 in the presentation](https://www.qax-camp.com.ua/courses/q6-basic/lessons/q6-defect/) or [screenshot](https://github.com/scholokov/qax-portal-2/assets/95171672/c705fbb7-8982-4cd9-ae33-6a5fb3158926)

**Environments**
Windows 10, Chrome 121.0.6167.140 (64 bit)
![Тест_ Defect – QAX Camp - Google Chrome 02 02 2024 15_37_34](https://github.com/scholokov/qax-portal-2/assets/95171672/a3608fb3-e336-4606-8c99-8fb9173ae2df)
![Image](https://github.com/scholokov/qax-portal-2/assets/95171672/5054e32e-6b41-4e54-b1e3-36024b0cfb5f)

**Comment by DmitriiNasonov** [comment]
#1920

**Comment by scholokov** [comment]
іноді це виноситься у окремен поле. тому прибрал. щоб не виникало питань :)

**Comment by scholokov** [comment]
тут перелічена саме обов'язкова інформація. оточення може бути присутнє, а може і у окремому полі. для уникнення зайвих питань прибрав. думаю що тут ок

**Comment by DmitriiNasonov** [comment]
I agree that the "Environment" block isn't necessarily present in the "Description" block, but the problem is that this information isn't in the lesson presentation and in the answer options in the test. It is in the video tutorial, but I’m not sure that everyone will watch the video when they get acquainted with the Defect or want to find the information they need quickly.
And if a person who doesn't watch the video (and, as a result, doesn't know that the “Environment” field isn't mandatory in the “Description” block and can be placed somewhere else) goes to take the test, he will have extra questions
